analytics_data_api/v0/views/learners.py

```diff
@@ -223,7 +233,13 @@ def get_queryset(self):
         # parameter values in `get_users_in_course`.
         params = {key: val for key, val in params.items() if val is not None}
         try:
-            return RosterEntry.get_users_in_course(self.course_id, **params)
+            users = RosterEntry.get_users_in_course(self.course_id, **params)
+            last_updated = {'date': None}
+            if len(RosterUpdate.get_last_updated()) == 1:
+                last_updated = RosterUpdate.get_last_updated()[0]
+            else:
+                logger.warn('RosterUpdate not found.')
+            return [{'learner': user, 'last_updated': last_updated} for user in users.execute()]
```

Comment:
Oh!  I tested this out quickly and saw that we were getting results for those fields (empty it turns out) and thought everything was working as expected.

I agree about moving the `execute()` call to the model--actually had intended on doing that.  I'll bring back `ElasticsearchDSLSearchSerializer` and see if I can figure out what went awry with the tests; oddly, they passed.